cni-plugin/Makefile

```diff
@@ -183,7 +203,7 @@ ut-datastore:
 	-v $(CERTS_PATH):/home/user/certs \
 	$(CALICO_BUILD) sh -c '$(GIT_CONFIG_SSH) \
 			cd  /go/src/$(PACKAGE_NAME) && \
-			ginkgo -cover -r -skipPackage pkg/install $(GINKGO_ARGS)'
+			ginkgo -cover -r -skipPackage pkg/install,containernetworking-plugins $(GINKGO_ARGS)'
```

Comment:
Do we really want to run unit tests that aren't "ours" as part of our CI? I would have suggested not, but maybe there is a reason?

Reply:
They are part of the skipPackage flag. The CI runs before the release can be made in the upstream repo, so I see no use for running them again. Also, the suite is flaky and sometimes has to run two or three times before it passes successfully.

Reply:
d'oh, yeah I shouldn't be reviewing PRs this early in the morning.